src/earn/EarnHome.tsx

```diff
@@ -273,17 +274,24 @@ export default function EarnHome({ navigation, route }: Props) {
             <EarnTabBar activeTab={activeTab} onChange={handleChangeActiveView} />
           </View>
         </Animated.View>
-        <PoolList
-          handleScroll={handleScroll}
-          listHeaderHeight={listHeaderHeight}
-          paddingBottom={insets.bottom}
-          displayPools={displayPools.filter((pool) =>
-            pool.tokens.some((token) =>
-              tokenList.map((token) => token.tokenId).includes(token.tokenId)
-            )
-          )}
-          onPressLearnMore={onPressLearnMore}
-        />
+        {zeroPoolsinMyPoolsTab ? (
```

Comment:
could we use https://reactnative.dev/docs/flatlist#listemptycomponent instead of our own boolean?

Reply:
ooo neat!

Reply:
It actually doesn't make it cleaner to use ListEmptyComponent because we also want to hide the 'learn more' footer when there are zero pools